app/helpers/application_helper.rb

```diff
@@ -85,6 +85,12 @@ def fraction_to_percentage_display(fraction, options = {precision: 2, significan
     end
   end
 
+  def formatted_month(month, include_nbsp = false)
```

Comment:
Since the `include_nbsp` parameter is never used when this function is called anywhere in the code, I wouldn't include it because that simplifies things.